package.json

```diff
@@ -151,10 +151,10 @@
     "chalk": "^2.4.2",
     "concurrently": "^5.1.0",
     "core-js": "^3.9.1",
+    "cors": "^2.8.5",
     "css-loader": "^6.7.3",
     "cssnano": "^6.0.0",
     "dotenv": "^8.0.0",
-    "dyson": "^4.1.0",
```

Comment:
`dyson` is outdated and archived: https://github.com/webpro/dyson#readme
It uses vulnerable `request` package which is deprecated and won't be fixed: https://github.com/request/request/issues/3142
Replaced it with custom mock server based on `express` with `cors` middleware